src/line-graph.js

```diff
@@ -34,7 +38,44 @@ export class LineGraph extends HTMLElement {
     this.renderGraph(this.parseData(data), svg);
   }
 
- 
+
+  /**
+   * Create set of distinct values
+   * @param {data} data 
+   */
+  maxValue(data) {
+    let arr = new Set()
+    data.forEach(element => {
+      element.datapoints.forEach(el => {
+        arr.add(el.value);
+      });
+    });
+    let min = Math.min(...[...arr]);
+    if (min === 0 && arr.size === 1) {
+      arr.add(-1);
+      arr.add(1);
+    } else if (arr.size === 1) {
+      let tenPer = (([...arr][0] * 10) / 100);
+      arr.add([...arr][0] - tenPer);
+      arr.add([...arr][0] + tenPer);
+    }
+    return [...arr];
+  }
+
+   /**
+   * Create set of time values
+   * @param {data} data 
+   */
+  maxTime(data) {
```

Comment:
Same for this function. This can be separate files or just one `helper.js` file that allows for exported classes.